chainstate/src/rpc/types/output.rs

```diff
@@ -39,7 +39,7 @@ pub enum RpcOutputValueIn {
     },
 }
 
-#[derive(Debug, Clone, serde::Serialize, rpc_description::HasValueHint)]
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, rpc_description::HasValueHint)]
```

Comment:
> *Out types deliberatly lack deserialzation as their purpose is to convert to user-friendly format.

We've discussed this and there doesn't seem to be a strong reason for "Rpc.*Out" types not to be deserializable. Their main difference from other types is redundancy (e.g. amounts contain both atoms and decimal values), so they are not suitable as inputs for rpc calls. But as outputs they may be consumed by other code, including rust code (the bridge uses them), so it makes sense for them to be deserializable.
So, I'm going to keep these structs deserializable and also change the functions that return `UtxoInfo` as `JsonValue`, so that they return `UtxoInfo` directly.